examples_utils/benchmarks/environment_utils.py

```diff
@@ -30,6 +32,52 @@ def get_mpinum(command: str) -> int:
     return mpinum
 
 
+def check_poprun_env_variables(benchmark_name: str, cmd: str) -> str:
+    """
+
+    """
+    # Python call is guarunteed to exist
+    using_python3 = "python3" in cmd
+    if using_python3:
+        poprun_call, python_call = cmd.split("python3")
+    else:
+        poprun_call, python_call = cmd.split("python")
+
+    # Check if $IPUOF_VIPU_API_PARTITION_ID is asked for (incorrect name),
+    # and replace this with $PARTITION (evaluated or not)
+    if "$IPUOF_VIPU_API_PARTITION_ID" in poprun_call:
+        poprun_call = poprun_call.replace("$IPUOF_VIPU_API_PARTITION_ID", os.getenv("PARTITION", "$PARTITION"))
```

Comment:
I was just doing a task to make those pick up the IPUOF variables.

https://graphcore.slack.com/archives/C03SX7GQCRH/p1660047473074119

Why are you stripping those out and looking for PARTITION?

Reply:
Ah I didnt know we were going the other way, ok I will remove this change then